.github/workflows/_check-user.yml

```diff
@@ -0,0 +1,72 @@
+name: Check whether the workflow owner can use ARC runners
+
+on:
+  workflow_call:
+    inputs:
+      user_name:
+        required: true
+        type: string
+        description: The name of the workflow owner.
+      curr_branch:
+        required: true
+        type: string
+        description: Current branch.
+
+    outputs:
+      workflow-type:
+        description: Type of runners to use
+        value: ${{ jobs.check-user.outputs.workflow-type }}
+
+jobs:
+  check-user:
```

Comment:
lets be very mindful of job names and file names. `check-user` is not clear what this job is actually doing.

Maybe something like `runner determinator` would make more sense

Reply:
I like the `runner determinator` version.